demisto_sdk/commands/init/tests/contribution_converter_test.py

```diff
@@ -42,6 +42,82 @@ def contribution_converter(request: FixtureRequest, tmp_path_factory: TempPathFa
     return create_contribution_converter(request, tmp_path_factory)
 
 
+@patch('demisto_sdk.commands.split_yml.extractor.get_python_version')
+@patch('demisto_sdk.commands.init.contribution_converter.get_content_path')
+def test_convert_contribution_zip_updated_pack(get_content_path_mock, get_python_version_mock, tmp_path):
+    """
+    Create a fake contribution zip file and test that it is converted to a Pack correctly.
+    The pack already exists, checking the update flow.
+
+    Args:
+        get_content_path_mock (MagicMock): Patch of the 'get_content_path' function to return the fake repo directory
+            used in the test
+        get_python_version_mock (MagicMock): Patch of the 'get_python_version' function to return the "3.7"
+        tmp_path (fixture): Temporary Path used for the unit test and cleaned up afterwards
+
+    Scenario: Simulate converting a contribution zip file, checking that the readme file is not updates in this scenario
```

Comment:
```suggestion
    Scenario: Simulate converting a contribution zip file
```

Reply:
Where do we see that the pack already exists before the `contribution_converter` executes `convert_contribution_to_pack()`?

Reply:
When we initialize the `ContributionConverter` class. I've added the `create_new=False` argument